specification/e-referrals-service-api.yaml

```diff
@@ -222,6 +222,8 @@ info:
   license:
     name: MIT
 servers:
+  - url: 'https://internal-dev-sandbox.api.service.nhs.uk/referrals-pr-181/FHIR'
```

Comment:
I'm assuming this is just for testing and will be removed before merging?

Reply:
Yes